runtime/compiler/il/J9Node.hpp

```diff
@@ -284,6 +284,8 @@ class OMR_EXTENSIBLE Node : public OMR::NodeConnector
    bool chkDontInlineUnsafePutOrderedCall();
    const char * printIsDontInlineUnsafePutOrderedCall();
 
+   bool isUnsafeCopyMemoryIntrinsic();
```

Comment:
This needs to be Doxygen documented as to why it exists and what is its purpose.

Reply:
Added brief in https://github.com/eclipse/openj9/compare/5b7e35a24c88400aebba54647550e37df536c44c..91cf3d3105d3cec00915a532186cdcbf5977262d